client/components/entry-form.jsx

```diff
@@ -92,7 +92,7 @@ export default class EntryForm extends React.Component {
                       <Image
                       style={{ height: 400, width: 400 }}
                       className='h-auto'
-                      src={this.state.file === null || this.state.file === '' ? 'https://t3.ftcdn.net/jpg/02/48/42/64/360_F_248426448_NVKLywWqArG2ADUxDq6QprtIzsF82dMF.jpg' : this.state.file}
+                        src={this.state.file === null || this.state.file === '' ? 'https://upload.wikimedia.org/wikipedia/commons/thumb/3/3f/Placeholder_view_vector.svg/681px-Placeholder_view_vector.svg.png' : this.state.file}
```

Comment:
I recommend downloading this image and putting it in your own `server/public` so you can use `/images/placeholder.svg` as your `src` attribute